### ISSUE TITLE
Nightly installation blocking through Microsoft Defender SmartScreen 

### ISSUE THREAD

**Issue by antoinekh** [mRemoteNG/mRemoteNG#2438]
Hello,

I can install stable and preview but I have an issue with nightly :

Is there a difference between nightly and stable (or preview)

## Expected Behavior
Installation ok

## Current Behavior
![image](https://github.com/mRemoteNG/mRemoteNG/assets/52744345/eb02b8c0-8b25-4d68-ae65-79df00940f08)

## Steps to Reproduce (for bugs)
Installation msi Nightly  v1.77.3.1784-NB

## Your Environment
![image](https://github.com/mRemoteNG/mRemoteNG/assets/52744345/c150d03f-bb46-4146-8a1a-f4cfbfae5dcf)

**Comment by Kvarkas** [comment]
thanks we know what, its due expirate certificate, you could ignore it, new release will include updated certificate already.

**Comment by antoinekh** [comment]
Unfortunately I can't expire it on my work laptop.

**Comment by Kvarkas** [comment]
that depends on windows settigs, just click on link and allow that not impact installation or work, just windows defender warning
certificate what was use to sigh others was valid then they was released so no such warning, but nb have it because certification already expired so we release it without signing as it still not for prod but kind of beta version.

**Comment by antoinekh** [comment]
Let me rephrase: my company's strict safety regulations prohibit me from passing this warning

**Comment by Kvarkas** [comment]
dont use nb for production, its not mean for that - next release of nb should not have this issue (soon)

**Comment by antoinekh** [comment]
I know I'm not using it now, but I need one of the latest features, ssh tunnel in particular.
The goal is to compare with my current solution mobaxterm.
But I'll wait for the next release
